rotkehlchen/chain/manager.py

```diff
@@ -1667,6 +1673,13 @@ def _add_protocol_balances(self) -> None:
                 eth_balances[address].assets[A_LQTY] += deposited_lqty
                 self.totals.assets[A_LQTY] += deposited_lqty
 
+        convex_module = self.get_module('convex')
+        if convex_module is not None and self.premium is not None:
+            convex_balances = convex_module.get_balances(addresses=self.queried_addresses_for_module('convex'))  # noqa: E501
+            for address, balance in convex_balances.items():
+                eth_balances[address].assets[A_CVX] += balance
+                self.totals.assets[A_CVX] += balance
+
```

Comment:
this is missing the balances for the different pools right? I only see CVX as asset here. For each pool you have the balances

![image](https://user-images.githubusercontent.com/5068010/176423173-c28fd513-9425-4d54-aaaf-6918d2a4b63a.png)

Reply:
Done